tests/lib/mock_command.py

```diff
@@ -2,12 +2,13 @@
 import os
 import sys
 
-if len(sys.argv) != 4:
+if len(sys.argv) < 4:
     sys.stderr.write("usage: {0} <stdout> <return_value> <stderr>".format(os.path.basename(__file__)))
 
 # W0632: Possible unbalanced tuple unpacking with sequence: left side has 3 label(s), right side has 0 value(s) (unbalanced-tuple-unpacking)
 # Disabled: Unpacking is balanced: there is a check for the length on line 5
-stdout, return_value, stderr = sys.argv[1:]  # pylint: disable=W0632
+# test adding extra arguments like pycharm helpers (/snap/pycharm-professional/412/plugins/python-ce/helpers/pydev/pydevd.py)
```

Comment:
weird issue observed in my runs, it took me long time to figure out this. As far as mocking args it will be present at the end in the args list

Reply:
could you add more details in the comments about what the issue was? I am not sure I fully understand it.